modules/op-installing-tkn-on-linux-on-ibm-power.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-linux-on-ibm-power"]
+
+= Installing {pipelines-title} CLI (tkn) on Linux on IBM Power Systems
+
+For Linux distributions, you can download the CLI directly as a `tar.gz` archive.
+
+.Procedure
+
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.12.0/tkn-linux-ppc64le-0.12.0.tar.gz[CLI].
```

Comment:
correct version is 0.13.1

Reply:
Is it 0.13.1 for both the folder and the tar file like this? 

https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.13.1/tkn-linux-ppc64le-0.13.1.tar.gz

Or only the tar?

Reply:
> Is it 0.13.1 for both the folder and the tar file like this?
> 
> https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.13.1/tkn-linux-ppc64le-0.13.1.tar.gz
> 
> Or only the tar?

both the folder